exo-go/src/application/docker_compose_builder.go

```diff
@@ -1,18 +1,18 @@
-package dockerSetup
+package application
 
 import (
 	"fmt"
 	"path"
 	"strings"
 
 	"github.com/Originate/exosphere/exo-go/src/config"
-	"github.com/Originate/exosphere/exo-go/src/docker_helpers"
+	"github.com/Originate/exosphere/exo-go/src/docker"
 	"github.com/Originate/exosphere/exo-go/src/types"
 	"github.com/Originate/exosphere/exo-go/src/util"
 )
 
-// DockerSetup renders docker-compose.yml file with service configuration
-type DockerSetup struct {
+// DockerComposeBuilder renders docker-compose.yml file with service configuration
```

Comment:
I think this comment is misleading since it doesn't render `docker-compose.yml`. It just gets the docker configs for the given service. I think I copied this from the livescript code. Sorry about that.